#
# Description: Enable the use of specific alias in the default config file like it's done in sendmail
# Forwarded: no
# Bug: no
# Author: Mjules <mjules AT free DOT fr>
#

--- dma.conf.orig	2010-12-25 16:09:31.177263621 +0100
+++ dma.conf	2010-12-25 16:09:49.226534728 +0100
@@ -10,7 +10,7 @@
 #PORT 25
 
 # Path to your alias file.  Just stay with the default.
-#ALIASES /etc/aliases
+ALIASES /etc/dma/aliases
 
 # Path to your spooldir.  Just stay with the default.
 #SPOOLDIR /var/spool/dma
